CONTRIBUTING.adoc

```diff
@@ -0,0 +1,55 @@
+= Contributing to Chaos Monkey for Spring Boot
+Thank you for willing to contribute! Every bit of help is highly welcome. 
+
+This document should help you get started if you want to contribute or hack on the code.
+
+**This is your first contribution to an open source project?** You can learn how to fork a project and how to create your first Pull Request from the excellent https://github.com/firstcontributions/first-contributions/blob/master/README.md[First Contributions] initiative.
+
+If you are unsure where to start, please head over to our https://github.com/codecentric/chaos-monkey-spring-boot/issues?q=is%3Aissue+is%3Aopen+label%3Agood-first-issue[Issue Tracker] where we have marked good beginner friendly issues. 
+
+
+== Using GitHub Issues
+We are using GitHub issues to track bugs, improvements and new features. If you have a general question on how to use Chaos Monkey for Spring Boot, please ask on Stack Overflow.
+
+
+=== Reporting Bugs
+Please help us to speed up problem diagnosis by providing as much information as possible. A small sample project that reproduces the error would be optimal for us. 
+
+
+=== Requests for Improvements & New Features
+New Pull Requests are highly welcome. However, before you provide us with a PR, please consider opening an issue using our GitHub issue tracker. This gives us the possibility to discuss the changes in details. If you're interested in an improvement or a new feature, try to provide us with your use case. 
+
+
+== Reporting Security Vulnerabilities
+If you found a security vulnerability please do not disclosure it publicly until we had a chance to fix it.
+
+Please do not use our issue tracker for security vulnerabilities you found. Instead use security@codecentric.de to disclose them responsibly. You can use our public PGP (Fingerprint: https://keyserver.ubuntu.com/pks/lookup?search=0x8921EEE5722496E4&fingerprint=on&op=index[`0x8921EEE5722496E4`]) key to encrypt your communication if needed.
+
+== Project Setup
+Fork and clone the repository 
+
+[NOTE]
+====
+To keep your fork in sync with the main project, link your `master` branch to the main repositories `master` branch. Do this by:
+
+[source, bash]
+----
+git remote add upstream https://github.com/codecentric/chaos-monkey-spring-boot.git
+----
+
+To fetch upstream changes switch to your master branch with `git checkout master` and then run `git pull upstream master`.
+====
+
+Chaos Monkey for Spring Boot is a Java based https://spring.io/projects/spring-boot[Spring Boot] plugin. In order to work with the code you need to have https://maven.apache.org/[Maven] installed. On Unix based systems we recommend using https://sdkman.io/[sdkman]. It helps you to manage your Java and Maven installations. On Windows you can fallback to installing Java SDK 8 or higher and the current Maven version directly.
```

Comment:
It sounds a bit picky, but I would preferr the Term "JDK" over "Java SDK" :neckbeard: